index.html

```diff
@@ -734,85 +673,128 @@ <h2 class="text-white">Digital Blockchain</h2>
+                <article
+                  class="liquid-lp liquid-blog-item liquid-blog-contents-inside liquid-blog-item-square liquid-blog-scheme-light round h-450">
+                  <figure class="liquid-lp-media round" data-responsive-bg="true">
+                    <a href="#">
+                      <img src="assets/demo/events/coders.jpg" alt="Lates Post" />
+                    </a>
+                  </figure>
+                  <div class="liquid-blog-item-inner round">
+                    <a href="#" class="liquid-overlay-link">Pre-CODERS</a>
+                    <header class="liquid-lp-header mt-auto">
+                      <h2 class="liquid-lp-title font-weight-bold h3 mb-0">
+                        <a href="#">Pre-CODERS</a>
+                      </h2>
+                      <div class="liquid-lp-details">
+                        <ul class="liquid-lp-category text-uppercase ltr-sp-175 font-weight-bold">
+                          <li><a href="#">Exciting Coding experience before the big battle</a></li>
+                        </ul>
+                      </div>
+                    </header>
                   </div>
-                </div>
+                </article>
               </div>
-              <div class="lqd-column col-md-3 col-sm-6">
-                <div
-                  class="iconbox iconbox-semiround iconbox-shadow iconbox-xl iconbox-filled iconbox-filled iconbox-filled-hover iconbox-light-onhover bg-ebony-clay pt-65 pb-65"
-                >
-                  <div class="iconbox-icon-wrap">
-                    <span class="iconbox-icon-container text-white">
-                      <i class="icon-basic_mail_open_text"></i>
-                    </span>
-                  </div>
-                  <div class="contents">
-                    <h3 class="text-white font-size-21 mb-25">
-                      Litecoin Miners
-                    </h3>
-                    <p class="text-fade-white-06">
-                      Bitcoin is designed around the idea cryptography.
-                    </p>
+              <div class="col-md-4 col-sm-6 masonry-item">
+                <article
+                  class="liquid-lp liquid-blog-item liquid-blog-contents-inside liquid-blog-item-square liquid-blog-scheme-light round h-450">
+                  <figure class="liquid-lp-media round" data-responsive-bg="true">
+                    <a href="https://hackersuop.github.io/sessions/">
+                      <img src="assets/demo/blog/blog-10.jpg" alt="Lates Post" />
+                    </a>
+                  </figure>
+                  <div class="liquid-blog-item-inner round">
+                    <a href="#" class="liquid-overlay-link">Competitive Programming Session</a>
+                    <header class="liquid-lp-header mt-auto">
+                      <h2 class="liquid-lp-title font-weight-bold h3 mb-0">
+                        <a href="#">Competitive Programming Session</a>
```

Comment:
Should be Plural : Sessions

Reply:
And also add the programming poster after today

Reply:
done